client/app/configuration/configuration.html

```diff
@@ -1,4 +1,4 @@
-<div class="container-fluid" id="main-container" data-ng-controller="UserCtrl">
```

Comment:
Pay attention that we are still using [`user.ovhSubsidiary`](https://github.com/ovh-ux/ovh-manager-web/pull/751/files#diff-402673dbf24274fbf7c8e7c006376f3aR11).

I'm not sure that you can remove `data-ng-controller="UserCtrl"`.

Reply:
Hi @antleblanc  Sorry for that. I had to close the PR and reopen it since it was a very old one. Ill take care of this.

Reply:
Please take a look now